src/main/java/com/gruzini/tennistico/controllers/ConfirmScoreController.java

```diff
@@ -0,0 +1,27 @@
+package com.gruzini.tennistico.controllers;
+
+import com.gruzini.tennistico.services.ConfirmScoreService;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import java.security.Principal;
+
+@Controller
+@RequestMapping("/dashboard/confirm-score")
```

Comment:
Nie kumam trochę dlaczego przez /dashbooard. Może zróbmy osobny endpoint do akcji? 
Bo przecież np. dołączasz do gry wcale nie ze strony dashboard. Tutaj akurat confirmujesz (możesz confirmować z dashboardu), ale możesz też ze strony z notyfikacjami.

Reply:
ta nazwa jest może trochę niefortunna, chodzi bardziej o to, że cała zawartość strony jak już jesteś zalogowany powinna startować od "/dashboard" albo jakkolwiek by to nazwać, "/site", "/main"

Reply:
/dashboard pochodzi z epoki kiedy zastanawialiśmy się nad endpointami z security. W obecnym stanie jestem jak najbardziej za tym żeby przerefactorować i zrobić po prostu /confirm-score

Reply:
Refactor na dużym refactorze